### PR TITLE
Align `storefront deploy` with react-storefront turborepo

### PR DESCRIPTION
**Pull request by 2can**

### CODE REVIEW

File: src/cli/storefront/deploy.ts [hunk 1]
```diff
@@ -39,12 +39,14 @@ export const handler = async () => {
     vercel,
     name,
     owner,
-    repoName
+    repoName,
+    'cd ../.. && npx turbo run build --filter="storefront"',
```
Comment (lkostrowski) [inline src/cli/storefront/deploy.ts:43]: This will probably not work on Windows, so better use path.join

**Review by lkostrowski — CHANGES_REQUESTED** [review]

File: src/cli/storefront/deploy.ts (hunk 1 above)
Comment (mmiszy) [inline src/cli/storefront/deploy.ts:43]:
```suggestion
    'cd ../.. && npx turbo run build --filter="storefront..."',
```

**Review by mmiszy — CHANGES_REQUESTED** [review]

File: src/cli/storefront/deploy.ts (hunk 1 above)
Comment (2can) [reply src/cli/storefront/deploy.ts:43]: @lkostrowski it's bulid command command for Vercel, there is no correlation to paths and Windows

https://vercel.com/docs/concepts/monorepos/turborepo#step-2:-set-the-build-command

Comment (2can) [reply src/cli/storefront/deploy.ts]: @mmiszy added, strangely it worked in Vercel without them

Comment (zaiste) [reply src/cli/storefront/deploy.ts]: @2can @mmiszy do we know why it works on Vercel without the `...`?

**Review by zaiste — APPROVED** [review]

**Review by lkostrowski — APPROVED** [review]

**Review by zaiste — APPROVED** [review]

**Pull request merged**